src/BloomExe/Program.cs

```diff
@@ -269,6 +269,7 @@ static void Main(string[] args1)
 			}
 			finally
 			{
+				Palaso.UI.WindowsForms.Reporting.MemoryManagement.CheckMemory(true, "Bloom finished and exiting", false);
```

Comment:
The third argument says whether it is allowed to warn the user.  Since it is false in this call, the user will never be warned as the program quits.